files/open-dyslexic/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "open-dyslexic",
+  "repo": "TheTannerRyan/open-dyslexic",
+  "files": {
+  	"basePath": "",
+    "include": ["*.eot", "*.otf", "*.svg", "*.ttf", "*.woff"]
```

Comment:
This is wrong. You can either fix it or remove update.json completely. I doubt this project will be updated again.